bin/stt_test/utils/SentenceChecker.py

```diff
@@ -0,0 +1,29 @@
+import spacy
+import subprocess
+import logging
+
+
+class SentenceChecker:
+
+    def __init__(self, model_name: str = "en_core_web_lg") -> None:
+        self.nlp = self._get_spacy_nlp(model_name)
+
+    def _get_spacy_nlp(self, model_name: str):
+        try:
+            return spacy.load(model_name)
+        except OSError:
+            logging.info("Spacy model not found. Downloading...")
```

Comment:
Nested try ... catch seems like a code smell, as well as catching OSError in that example. Can't we check if the model is downloaded before trying to load? 
`spacy.util.is_package(model_name)`

Reply:
I would need to check where it is supposed to be. I'm guessing it will probably be in .venv in site-packages. I feel like we should only go digging there as a last resort. How about moving just one try except to the _download_model function?

Reply:
Have you tried using **spacy.util.is_package(model_name)**?

Reply:
Great discovery!